doc-content/jbpm-docs/src/main/asciidoc/BPMN2/con-timer.adoc

```diff
@@ -0,0 +1,67 @@
+[id='con-timer_{context}']
+= Timers
+
+You can use timers to trigger a logic after a certain period or to repeat specific actions at regular intervals. Timers wait for a predefined amount of time before triggering once or repeatedly.
+
+== Configuration of timers with delay and period
+
+You can set a timer with delay and a certain period. The delay specifies the waiting time after the node activation, and the period defines the time between the subsequent trigger activation. The period value `0` results in a one-shot timer. You can specify the delay and period expression in `[#d][#h][#m][#s][#[ms]]` form, indicating the number of days, hours, minutes, seconds, and milliseconds (default). For example, the expression `1h` indicates one hour waiting time before triggering the timer again.
+
+== Configuration of timers with ISO-8601 date format
+
+You can configure timers with a valid ISO-8601 date format that supports both one-shot timers and repeatable timers. You can define timers as date and time representation, time duration, or repeating intervals. For example:
```

Comment:
Question - does valid mean a future date or does it mean "as per the ISO-8601 date format".

If the latter, then it is not required and "valid" can be removed.

Reply:
Done, removed valid